docs/translating.md

```diff
@@ -0,0 +1,24 @@
+# Translating the Application
+
+## How Languages Are Defined
+
+1. Add the new language code and language name to the map `supportedLanguages` in [/src/i18n/baseConfig.ts](../src/i18n/baseConfig.ts)
+2. Create a folder in [/public/locales/](../public/locales) with the language code as its name.
+3. Add a new file, `translations.json`, to the new folder.
+4. Translate all the entries.  If you prefer to use the English version just set the value to `null`.
+
+   Example file: [/public/locales/ja-JP/translations.json](../public/locales/ja-JP/translations.json)
+
+## Updating Existing Translations
+
+When making changes to the base (English) language file follow the ?
+
+### New Entry
+1. Create entry in `en-US/translations.json`
+2. Add an entry to all other `translation.json` files with a value of `null`
```

Comment:
This phrasing is a bit ambiguous - it could mean only add an entry if there is a value of null in the other translation.json files. (Was a bit confused when I read it, even though with an understanding of what's going on that other interpretation doesn't make sense. Hopefully this makes it more clear though)
```suggestion
2. Add an entry to all other `translation.json` files. Set the value to be `null` (which means it will default to using the English version)
```

Reply:
Addressed in 3adc028777c037d5e972b557ebd0f75e63fffbec.